package_components.toml

```diff
@@ -1,3 +1,29 @@
+[Cairo_jll]
+"1.14.12+0" = {cairo = "1.14.12"}
+"1.14.12+1" = {cairo = "1.14.12"}
+"1.14.12+2" = {cairo = "1.14.12"}
+"1.14.12+3" = {cairo = "1.14.12"}
+"1.14.12+4" = {cairo = "1.14.12"}
+"1.14.12+5" = {cairo = "1.14.12"}
+"1.14.12+6" = {cairo = "1.14.12"}
+"1.16.0+0" = {cairo = "1.16.0"}
+"1.16.0+1" = {cairo = "1.16.0"}
+"1.16.0+2" = {cairo = "1.16.0"}
+"1.16.0+3" = {cairo = "1.16.0"}
+"1.16.0+4" = {cairo = "1.16.0"}
+"1.16.0+5" = {cairo = "1.16.0"}
+"1.16.0+6" = {cairo = "1.16.0"}
+"1.16.1+0" = {cairo = "1.16.0"}
+"1.16.1+1" = {cairo = "1.16.0"}
+"1.16.1+2" = {cairo = "1.16.0"}
+"1.18.0+0" = {cairo = "1.18.0"}
+"1.18.0+1" = {cairo = "1.18.0"}
+"1.18.0+2" = {cairo = "1.18.0"}
+"1.18.2+0" = {cairo = "1.18.2"}
+"1.18.2+1" = {cairo = "*"}
```

Comment:
```suggestion
"1.18.2+1" = {cairo = ["1.18.2","1.18.4"]}
```

This commit lands about a dozen commits ahead of the tagged 1.18.2 with some significant changes. Vulnerabilities against _both_ 1.18.2 and 1.18.4 (the subsequent release) might apply.